bundler/lib/dependabot/bundler/package_manager.rb

```diff
@@ -44,9 +43,14 @@ def initialize(version)
       def deprecated?
         deprecated_versions.include?(version)
       end
+
       sig { override.returns(T::Boolean) }
       def unsupported?
-        !deprecated? && version < supported_versions.first
+        # Check if the feature flag for Bundler v1 unsupported error is enabled.
+        return false unless Dependabot::Experiments.enabled?(:bundler_v1_unsupported_error)
```

Comment:
This feature flag is going to be used to enable not supported version for bundler v1 when the date arrives.